.vscode/launch.json

```
@@ -8,8 +8,8 @@
       "name": "Hyperion",
       "type": "python",
```

Would you mind changing this to `debugpy` to force the usage of the new Python debugger? The current setting was recently deprecated, and I think this PR would be a great occasion to change it.
https://devblogs.microsoft.com/python/python-in-visual-studio-code-february-2024-release/#python-debugger-extension-installed-by-default

```suggestion
      "type": "debugpy",
```